### ISSUE TITLE
No tasks detected when not use wrapper

### ISSUE THREAD

**Issue by xiaoyao9184** [microsoft/vscode-gradle#819]
Extension Name: vscode-gradle
Extension Version:  3.5.2
OS Version: Windows 10 20H2 19042.685
VSCode version: 1.52.2

My project does not use gradle wrapper.
 
Try use
```groovy
"gradle.autoDetect": "off"
```
but not show gradle task view 

When i use `vscode-gradle` and `Java Extension Pack` together with specified local installation directory,
```json
    "java.import.gradle.wrapper.enabled": false,
    "java.import.gradle.home": "D:\\Bin\\gradle-5.6",
```
"Gradle Tasks" output this 

**Output from "Gradle Tasks"**
```log
[debug] [gradle-server] Starting server
[debug] [gradle-server] Gradle Server cmd: c:\Users\xiaoyao9184\.vscode\extensions\richardwillis.vscode-gradle-3.5.2\lib\gradle-server.bat 6044
[info] [gradle-server] Server started, listening on 6044
[info] Gradle client connected to server
[debug] Refreshing tasks
[error] [gradle-server] Gradle version is required
[error] Error getting build for e:\WorkProjects\********************: Gradle version is required
[info] Found 0 tasks
```

**Does the bug still exist if you disable all other extensions?**
Yes without wrapper
No with `Java Extension Pack` and disable it wrapper mode.

**Comment by badookey** [comment]
I am having similar issues where i do not have a wrapper and need to work in offline mode.

I have the below settings enabled
```
"java.import.gradle.offline.enabled": true,
"java.import.gradle.wrapper.enabled": false,
"java.import.gradle.version": "6.0.1",
"java.import.gradle.home": "/opt/gradle/gradle-6.0.1/bin/gradle",
"gradle.nestedProjects": true,
"gradle.debug": true,
"gradle.autoDetect": "off",
```

No tasks appear in the 'gradle tasks' window, and upon clicking 'run a gradle build' and inputting 
`build -x test`
I get the following error:
`org.gradle.tooling.GradleConnectionException: Could not install Gradle distribution from 'https://services.gradle.org/distributions/gradle-6.0.1-bin-zip.`

I can run the same gradle command on the command line fine, as well as if i configure it as a vscode task

**Comment by CsCherrYY** [comment]
fixed in #922. Close it now. If you find this issue still exists in `3.7.0`, feel free to reopen it.
